src/scaffold/store.ts

```diff
@@ -45,17 +45,10 @@ export const useStore = create<Store>((set) => {
 
       case 'sendTemplatesOptions':
         const templatesOptions = message.templatesOptions;
-        if (!templatesOptions) {
-          dispatch(set, (state) => ({
-            ...state,
-            templateOptions: [],
-          }));
-          return;
-        }
-        const options = Object.keys(templatesOptions).map((key) => [key, templatesOptions[key]]);
         dispatch(set, (state) => ({
           ...state,
-          templateOptions: options,
+          isLoading: false,
+          templateOptions: templatesOptions ? templatesOptions : {},
```

Comment:
Either use `templateOptions || {}` or `templateOptions ?? {}`. But not this approach - its repetitive.